examples/experimental/i3s-17-and-debug/tile-debug.js

```diff
@@ -20,6 +20,11 @@ const REFINEMENT_TYPES = {
 
 const FLOAT_VALUES_FIXED_COUNT = 3;
 
+/**
+ * Return short tile info
+ * @param {object} tileHeader
```

Comment:
We should create a typescript type for the TileHeader

Reply:
Ok. Will do in further PR.